live-examples/js-examples/expressions/expressions-nullishcoalescingoperator.html

```diff
@@ -0,0 +1,11 @@
+<pre>
+<code id="static-js">
```

Comment:
The newline here means the example starts with a blank line. So this should be:

```
 <code id="static-js">const foo = null ?? "default string";
```

(I know, this is really confusing and we ought to fix it.)